pkg/generators/openapi.go

```diff
@@ -671,7 +698,9 @@ func (g openAPITypeWriter) generateSliceProperty(t *types.Type) error {
 	case types.Struct:
 		g.generateReferenceProperty(elemType)
 	case types.Slice, types.Array:
-		g.generateSliceProperty(elemType)
+		if err := g.generateSliceProperty(elemType); err != nil {
+			return err
+		}
 	default:
```

Comment:
do we miss the map case here? I.e. arrays of maps.

Reply:
@stts done. Added test for more coverage.

Reply:
sttts ;-)